tests/dummy/app/controllers/besluit-sample.ts

```diff
@@ -355,7 +360,177 @@ export default class BesluitSampleController extends Controller {
+            action: (controller: SayController) =>
+              insertDescriptionAtCursor(controller, this.intl),
+            buttonTitle: 'Insert description',
+          },
+          {
+            shaclRule:
+              'https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-motivering-validation',
+            action: (controller: SayController) =>
+              insertMotivationAtCursor(controller, this.intl),
+            buttonTitle: 'Insert motivation',
+          },
+          {
+            shaclRule:
+              'https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-article-container-validation',
+            action: (controller: SayController) =>
+              insertArticleContainerAtCursor(controller, this.intl),
+            buttonTitle: 'Insert article container',
+          },
+        ],
+        documentShape: `
+        @prefix sh:      <http://www.w3.org/ns/shacl#> .
+@prefix qb:      <http://purl.org/linked-data/cube#> .
+@prefix lblodBesluit:	<http://lblod.data.gift/vocabularies/besluit/> .
+@prefix ext:	<http://mu.semte.ch/vocabularies/ext/> .
+<https://data.vlaanderen.be/shacl/besluit-publicatie#BesluitShape>
+	a sh:NodeShape ;
+	sh:targetClass <http://data.vlaanderen.be/ns/besluit#Besluit> ;
+	sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-description-validation>;
+    sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-article-container-validation>;
+	sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-short-title-validation>;
+    sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-title-validation>;
+	sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-language-validation>;
+    sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-article-validation>;
+    sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-citation-validation>;
+    sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-motivering-validation>;
+	sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-publication-date-validation>;
+    sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-date-no-longer-in-force-validation>;
+	sh:property <https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-first-date-entry-in-force-validation>;
+	sh:closed false .
+
+<https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-description-validation> sh:name "beschrijving" ;
+		sh:description "Een beknopte beschrijving van het besluit." ;
+		sh:path <http://data.europa.eu/eli/ontology#description> ;
+		 sh:or (
+      [
+        sh:datatype <http://www.w3.org/2001/XMLSchema#string>;
+      ]
+      [
+        sh:datatype <http://www.w3.org/1999/02/22-rdf-syntax-ns#langString>;
+      ]
+    );
+    sh:minCount 1 ;
+		sh:maxCount 1 ;
+        sh:resultMessage "Het besluit mag maximaal één beschrijving hebben.";
```

Comment:
```suggestion
        sh:resultMessage "Het besluit moet exact één beschrijving hebben.";
```

Reply:
"max 1" is not accurate, it's "exactly 1"
this is otherwise a very weird message when you have a missing description